#132.java

```diff
@@ -0,0 +1,48 @@
+/**
+132. Palindrome Partitioning II
+
+Given a string s, partition s such that every substring of the partition is a palindrome.
+
+Return the minimum cuts needed for a palindrome partitioning of s.
+
+For example, given s = "aab",
+Return 1 since the palindrome partitioning ["aa","b"] could be produced using 1 cut.
+**/
+
+
+/**
+ create a int array minCut to cache the minimum cuts needed for a palindrome partition for string from position j to the
```

Comment:
minCut[j] stores the minimum cuts needed for a palindrome partition for substring from index j to the end of string, thus minCut[0] stands for the minimum cuts needed for a palindrome partition for substring starting with index 0, the entire string.